### PR TITLE
Add ansible-awx icon

### LINKED ISSUE #7946: AWX
### Brand Name

AWX

### Website

https://github.com/ansible/awx-logos

### Popularity Metric

https://github.com/ansible/awx

### Official Resources for Icon and Color

https://github.com/ansible/awx-logos/raw/master/awx/ui/client/assets/favicon.svg

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by yqlbu**
**Issue:** closes #7946 

**Similarweb rank:**

https://github.com/ansible/awx

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

The link below is the official logo source for `ansible-awx`

https://github.com/ansible/awx-logos/raw/master/awx/ui/client/assets/favicon.svg

### CODE REVIEW

**Comment by yqlbu** [comment]
I found it hard to do. Please help.

**Comment by LitoMore** [comment]
Let's use `AWX` text only, without the background shape.

**Comment by yqlbu** [comment]
> Resolved in [9652576](https://github.com/simple-icons/simple-icons/pull/7947/commits/96525768f8fe2736904f79580b4b721b98696419)

File: _data/simple-icons.json [hunk 1]
```diff
@@ -742,6 +742,11 @@
             "hex": "EE0000",
             "source": "https://www.ansible.com/logos"
         },
+        {
+            "title": "Ansible AWX",
+            "hex": "2F8CCC",
+            "source": "https://github.com/ansible/awx-logos/blob/master/awx/ui/client/assets/favicon.svg"
```
[the remark this answers is not in the document]
Comment (yqlbu) [reply _data/simple-icons.json]: Resolved in [eed7850](https://github.com/simple-icons/simple-icons/pull/7947/commits/eed78504ebafdf5456ab219b6ce8be76a3e2e457)

**Comment by LitoMore** [comment]
@yqlbu The failing CI needs to be fixed. Here is our [contributing guidelines](https://github.com/simple-icons/simple-icons/blob/068d158e7dcad4dcfa6d5d9c68d54d431d70acfe/CONTRIBUTING.md).

**Pull request closed without merging**